frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +76,7 @@ class Sidebar extends PureComponent<SidebarProps, State> {
     )
     this.state = {
       collapsedSidebar: Sidebar.shouldCollapse(props, this.mediumBreakpointPx),
+      sidebarWidth: window.localStorage.getItem("sidebarWidth") || "336",
```

Comment:
Maybe have a `const minWidth = "336"` and then do `window.localStorage.getItem("sidebarWidth") }} || minWidth`

Reply:
Makes sense, added a `this.minWidth` 👍🏼